app/components/download-graph.hbs

```diff
@@ -1,3 +1,37 @@
+<div local-class="toggle-stacked">
```

Comment:
could you move the dropdown out of the `DownloadGraph` component and into the parent component or route template using something like `<DownloadGraph @stacked={{this.stackedGraph}} />`? :)

Reply:
Sure!